src/khoj/interface/web/base_config.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🦅</text></svg>">
+        <title>Khoj - Configure App</title>
```

Comment:
nit: consider setting title to `Khoj - Configure` to keep title format same as `Khoj - Chat` and `Khoj - Search`?